include/aspect/geometry_model/two_merged_boxes.h

```diff
@@ -195,6 +195,15 @@ namespace aspect
         parse_parameters (ParameterHandler &prm) override;
 
       private:
+        /**
+         * Wether to make the grid by gluing together two boxes, or just
+         * use one chunk to make the grid. Using two grids glued together
+         * is a safer option, since it forces the the boundary conditions
+         * to be always applied to the same depth, but using one grid allows
+         * for a more flexable usage of the adaptive refinement.
```

Comment:
```suggestion
         * Whether to make the grid by gluing together two boxes, or just
         * use one chunk to make the grid. Using two grids glued together
         * is a safer option, since it forces the boundary conditions
         * to be always applied to the same depth, but one unified grid allows
         * for a more flexible usage of the adaptive refinement.
```